<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Login_Screen</class>
 <widget class="QWidget" name="HNS_Login_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="LE_Username">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>28</y>
     <width>500</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141;border: 3px solid #414141;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>USERNAME</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Batt">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>88</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>Battery 13.0 V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Sol">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>128</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>Sol 0.0A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_LogIn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>218</y>
     <width>312</width>
     <height>95</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato Heavy</family>
     <pointsize>33</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#EB6123;
color:#FFFFFF;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_SolV">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>168</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>Sol 0.0V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Sign">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>48</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>SMC-4000</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_NTCIP_Waiting">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>273</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>NTCIP Database
loading, please wait</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Version">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>8</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>Version 1.0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_Password">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>128</y>
     <width>500</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141;border: 3px solid #414141;border-radius:10px;</string>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGraphicsView" name="GV_Logo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>28</y>
     <width>177</width>
     <height>177</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;</string>
   </property>
  </widget>
  <widget class="HNS_Date_Widget" name="LBL_Date" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="HNS_Clock_Widget" name="LBL_Time_2" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Debug">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#414141</string>
   </property>
   <property name="text">
    <string>DEBUG</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNS_Date_Widget</class>
   <extends>QWidget</extends>
   <header>hns_date_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HNS_Clock_Widget</class>
   <extends>QWidget</extends>
   <header>hns_clock_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
